soccer/src/soccer/radio/radio.cpp

```diff
@@ -39,7 +39,7 @@ Radio::Radio()
     }
 
     alive_robots_pub_ =
-        create_publisher<rj_msgs::msg::AliveRobots>("/alive_robots", rclcpp::QoS(1));
```

Comment:
For some reason base Radio class (used by network_radio) and sim_radio used different topic names for alive robots. I went with the one used by sim.